src/sentry/web/frontend/accounts.py

```diff
@@ -122,6 +122,14 @@ def recover_confirm(request, user_id, hash):
 
 @login_required
 def start_confirm_email(request):
+    from sentry.app import ratelimiter
+
+    if ratelimiter.is_limited(
+        'auth:confirm-email:{}'.format(request.user.id),
+        limit=10, window=60,  # 10 per minute should be enough for anyone
+    ):
+        return HttpResponse('stahp', status=429)
```

Comment:
i agree someone probably is abusing things intentionally here, but we should show a real error message